CommunityFridgeMapApi/tests/unit/test_tag.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from dependencies.python.db import Tag
+
+class DynamoDbMockPutItem():
+    def __init__(self):
+        pass
+
+    def put_item(self, TableName=None, Item=None, ConditionExpression=None):
+        pass
+
+class TagTest(unittest.TestCase):
+
+    def test_is_valid_tag_name(self):
+        tag = Tag(tag_name='test-tag', db_client=None)
+        self.assertTrue(tag.is_valid_tag_name('test-tag')[0])
+        self.assertEqual(tag.is_valid_tag_name('test-tag')[1], '')
+        tag = [
+            Tag(tag_name='', db_client=None),
+            Tag(tag_name=None, db_client=None),
+            Tag(tag_name='ab', db_client=None),
+            Tag(tag_name='test&tag', db_client=None),
+            Tag(tag_name='TEST-tag_tag123xyui1234567890__--', db_client=None)
+        ]
+        expected_false = [
+            tag[0].is_valid_tag_name('')[0],
```

Comment:
You can simply use `Tag.is_valid…` since this is a static method

Reply:
Why would lines get longer? You are replacing `tag[i]` with `Tag`

Reply:
Okay- I like it that way. It looks cleaner.